app/components/primer/image_crop.rb

```diff
@@ -19,9 +19,7 @@ class ImageCrop < Primer::Component
     #
     # @example Cropper with a custom loader
     #   <%= render(Primer::ImageCrop.new(src: "https://github.com/koddsson.png", rounded: false)) do |cropper| %>
-    #     <% cropper.loading(style: "width: 120px") do %>
-    #       <img src="spinner.gif />
-    #     <% end %>
+    #     <% cropper.loading(style: "width: 120px", tag: :img, src: "spinner.gif") %>
```

Comment:
What do you think about using the SpinnerCcomponent in this example?

Reply:
I wanted to showcase a different spinner since we show a SpinnerComponent if the slot is not invoked. I don't mind changing the example, but I just thought it would be not very clear.